building Mangohud-0.8.0 -rc2 from the DFSG source fails

Building Mangohud-v0.8.0-rc2 from the "DFSG" source fails with the compiler complaining about the file 'nvml.h' missing. 
BTW, compiling from the git or the full source works fine.

If it is of any help, this is the compiler error message:

`g++ -Isrc/libMangoHud.a.p -Isrc -I../src -I../include -Isubprojects/imgui-1.89.9 -I../subprojects/imgui-1.89.9 -I../subprojects/imgui-1.89.9/backends -I../subprojects/Vulkan-Headers-1.2.158/include -Isubprojects/implot-0.16 -I../subprojects/implot-0.16 -I. -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -fvisibility=hidden -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c++17 -Werror=missing-declarations -Werror=return-type -Wno-unused-parameter -fno-math-errno -fno-trapping-math -Wno-non-virtual-dtor -Wno-missing-field-initializers -O2 -g -pipe -Wformat -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -fomit-frame-pointer -m32 -march=i686 -msse2 -mtune=generic -mfpmath=sse -mstackrealign -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection=full -fPIC -pthread -DSPDLOG_SHARED_LIB -DSPDLOG_FMT_EXTERNAL '-DIMGUI_API=__attribute__((visibility("default")))' -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS '-DPACKAGE_VERSION="v0.8.0"' -DSPDLOG_COMPILED_LIB -DSPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_DEBUG -D_GNU_SOURCE -DHAVE_PTHREAD -DUSE_GCC_ATOMIC_BUILTINS -DHAVE_TIMESPEC_GET -DHAVE___BUILTIN_BSWAP32 -DHAVE___BUILTIN_BSWAP64 -DHAVE___BUILTIN_CLZ -DHAVE___BUILTIN_CLZLL -DHAVE___BUILTIN_CTZ -DHAVE___BUILTIN_EXPECT -DHAVE___BUILTIN_FFS -DHAVE___BUILTIN_FFSLL -DHAVE___BUILTIN_POPCOUNT -DHAVE___BUILTIN_POPCOUNTLL -DHAVE___BUILTIN_UNREACHABLE '-DMANGOHUD_ARCH="32bit"' -DHAVE_X11 -DHAVE_WAYLAND -DHAVE_DBUS -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_WAYLAND_KHR -MD -MQ src/libMangoHud.a.p/nvidia.cpp.o -MF src/libMangoHud.a.p/nvidia.cpp.o.d -o src/libMangoHud.a.p/nvidia.cpp.o -c ../src/nvidia.cpp
../src/nvidia.cpp:1:10: fatal error: nvml.h: No such file or directory
    1 | #include "nvml.h"
      |          ^~~~~~~~
compilation terminated.
`

Regards.

--- a/src/nvidia.cpp
+++ b/src/nvidia.cpp
@@ -1,4 +1,6 @@
+#ifdef HAVE_NVML
 #include "nvml.h"
+#endif
 #include "hud_elements.h"
 #include "logging.h"
 #include "string_utils.h"

